Public/Src/Engine/Scheduler/IQueryablePipDependencyGraph.cs

```diff
@@ -111,5 +111,10 @@ public interface IQueryablePipDependencyGraph
         /// Returns the seal directory pip that corresponds to the given directory artifact
         /// </summary>
         Pip GetSealedDirectoryPip(DirectoryArtifact directoryArtifact, PipQueryContext queryContext);
+
+        /// <summary>
+        /// Returns whether pip <paramref name="to"/> is reachable from pip <paramref name="from"/>.

```

Comment:
>reachable [](start = 57, length = 9)

nit: I'd expand the comment, I find 'reachable' here lacking 'explaining power'. Maybe add the comment you added in the analyzer , i.e., that pip 'from' is executed before pip 'to'?